cloudinit/config/cc_mounts.py

```diff
@@ -223,42 +223,86 @@ def suggested_swapsize(memsize=None, maxsize=None, fsys=None):
     return size
 
 
+def get_fstype(path):
```

Comment:
Do we need this function call that we are just grabbing the 1 index of get_mount_info? We might be able to just move the comment down above to the callsite instead of paying the extra cost of a function call.